src/pages/resultsView/pathwayMapper/ResultsViewPathwayMapper.tsx

```diff
@@ -30,6 +29,7 @@ import 'cytoscape-panzoom/cytoscape.js-panzoom.css';
 import 'cytoscape-navigator/cytoscape.js-navigator.css';
 import 'react-toastify/dist/ReactToastify.css';
 import styles from './pathwayMapper.module.scss';
+import PathwayMapper, { ICBioData } from 'pathway-mapper';
```

Comment:
it's interesting that if you only use these as types, then the library doesn't get bundled.  which is what we want.

Reply:
interesting. I guess that makes sense since types are not included in runtime